### ISSUE TITLE
Changes to published views are not reflected in the sourcecode

### ISSUE THREAD

**Issue by jmartsch** [spatie/statamic-responsive-images#9]
Publishing the views via `php artisan vendor:publish` seem to work fine. The files are copied.

But after modifying the HTML files in resources/views/vendors/statamic-responsive-images/responsiveImage.antlers.html the output is still the default. 

I only wrote the word "test" into this file but still get the normal default output instead of "test".

Here is the tag I use:
```
<figure>
        {{ image }}
            {{ if extension !== 'gif' }}
                {{ responsive:url data-src="{{ url }}" glide:width="768" alt="{alt}" class="{class} mx-auto lazyload" placeholder="false" }}
            {{ else }}
                <picture>
                    <img src="{{ url }}" alt="{title}" class="mx-auto lazyload">
                </picture>
            {{ /if }}
        {{ /image }}
    {{ if caption }}
        <figcaption class="block text-base text-gray-600 text-center max-w-sm mx-auto">{{ caption }}</figcaption>
    {{ /if }}
</figure>
```

**Comment by pryley** [comment]
I have the same issue, I am also unable to publish the views:

When I run `php artisan vendor:publish` it throw the following error:

![image](https://user-images.githubusercontent.com/134939/89597295-56332680-d81f-11ea-864a-29f8f715c500.png)

However, the command creates the folder in `/resources/views/vendor/statamic-responsive-images`, but it is empty.

Copying over the files to this folder manually and clearing the stache/cache does nothing, any custom changes are not used.

**Comment by riasvdv** [comment]
Should be fixed in v1.2.3!

**Comment by jmartsch** [comment]
Fix confirmed. It works 👍
